examples/hello_env/BUILD

```diff
@@ -0,0 +1,26 @@
+load(
+    "@io_bazel_rules_rust//rust:rust.bzl",
+    "rust_binary",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+rust_binary(
+    name = "hello_env",
+    srcs = ["src/main.rs"],
+    deps = ["@examples//hello_lib"],
+    rustc_env_files = [":generate_rustc_env_file"],
```

Comment:
What do you think about creating a `tests` directory and moving `hello_env` there? I'd expect `examples` directory to serve as a documentation, inspiration, for the user, and `hello_env` might be more an integration test than an example.

For an actual example it would be cool to see how you typically use `--stamp`.

Orthogonally, would you be fine with naming the directory `rustc_env_files` or something more precise?

Reply:
Ooh, TIL there's a `test` directory! Moved to `test` and renamed, but also added an example :)